index.html

```diff
@@ -1,72 +1,225 @@
-    <meta name="title" content="The Stargate Rewritten Project">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/5.0.2/css/bootstrap.min.css" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/font-awesome-line-awesome/css/all.min.css" />
-    <link rel="stylesheet" href="css/global.css" />
-    <link rel="stylesheet" href="css/style.css" />
-</head>
-
-<body>
-    <section id="hero" class="d-flex flex-column align-items-center justify-content-center">
-        <h2 class="hero-title display-2 text-uppercase mt-5">
-            <span draggable="true" data-highlightable="1" id="i1ycn">Stargate </span>rewritten
-        </h2>
-        <p class="hero-subtitle display-6">The original — and still the best — transportation solution!</p>
-        <div id="icbq6">A MineCraft plugin for the Bukkit, Sponge, and PMMP ecosystems.<br/></div>
-        <div class="hero-buttons">
-            <a href="https://sgrewritten.org/download"><button class="btn btn-primary me-2" id="ovrd">
-					<span class="fas me-2 fa-download"></span>Downloads
-				</button></a>
-        </div>
-        <div class="hero-buttons">
-            <a href="https://sgrewritten.org/wiki"><button class="btn btn-outline-secondary" id="b1">
-					<span class="fas me-2 fa-book" id="i1"></span>Documentation
-				</button></a>
-            <a href="https://sgrewritten.org/discord"><button class="btn btn-secondary" id="b2">
-					<span class="fas me-2 fa-comments" id="i2"></span>Support
-				</button></a>
-            <a href="https://sgrewritten.org/bukkitsource"><button class="btn btn-outline-secondary" id="b3">
-					<span class="fas me-2 fa-code" id="i3"></span>Source Code
-				</button></a>
-        </div>
-        <div class="hero-image p-4 shadow-sm rounded" id="otovrd"><img src="https://i.imgur.com/6BMpZKc.png" alt="Gramateria" id="iw1wr" /></div>
-    </section>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/5.0.2/js/bootstrap.min.js"></script>
-    <script src="js/global.js"></script>
-    <script src="js/script.js"></script>
-</body>
-
-<footer>
-    <div style="background-color: rgb(0,0,0, 0.7); padding: 10px 0px 0px 0px !important; bottom: -10px; position: fixed; text-align: center; width:100%;"><span style="color:LightGray;"><b>Temporary website; replacement coming soon!</b></span><br><small><i><p style="color:LightGray;">Background image provided by <a style="color:LightGray;" href="https://unsplash.com/@josiahgascho">Josiah Gascho</a> via <a style="color:LightGray;" href="https://unsplash.com/photos/YnICeUmhzCA">Unsplash</a></p></i></small></div>
-</footer>
-
-</html>
+<!DOCTYPE html>
+<html lang="zh-TW">

```

Comment:
We do not currently have a way to localise the website.
As such, given that it's text is in English, we should probably let people's browsers translate them.
Declaring that this page is in Chinese would likely just confuse people's browsers.